docs/docs/rest-api/public/api/v2/types/volumes.raml

```diff
@@ -58,6 +58,11 @@ types:
         minimum: 0
         description: |
           For `mount` mesos disk resources, the optional maximum size of an exclusive mount volume to be considered.
+      profileName?:
+        type: string
```

Comment:
Should this be an enum?

Reply:
Why? I do not get what you mean here. A profile name is an optional arbitrary non-empty string.